component-photo-gallery/src/Organisms/ImageList.js

```diff
@@ -1,12 +1,17 @@
 import React, { Component } from 'react';
+import ImageItem from '../Molecules/ImageItem'
 
 export default class ImageList extends Component {
     render() {
+        const allImages = this.props.images.map(image => {
+            return (
+                <ImageItem image = {image} />
```

Comment:
so in here in your map you will want to add something like this:
<ImageItem   key={yourdata.title}   image = {image} />

this will give each card its own "key"